src/screens/SelectMultiple.js

```diff
@@ -254,6 +254,10 @@ const SelectMultiplePage = () => (
 
         <Property name="messages">
           <Description>Custom messages.</Description>
+          <Description>
+            To affect all SelectMultiple instances, use messages prop on Grommet
```

Comment:
Is it possible to include an Anchor/ReactRouter link in this?

Like <Anchor label="messages" href="/grommet#messages" />

Reply:
good question let me see

Reply:
added!